src/core/core.controller.js

```diff
@@ -226,6 +247,26 @@ class Chart {
 		return me;
 	}
 
+	/**
+	 * @private
+	 */
+	initializePlatform(item, config) {
```

Comment:
We usually prefix private methods with underscore

Can we call this `canvas` instead of `item` as well?